Makefile

```diff
@@ -1,5 +1,4 @@
 IMAGE := tfsec/tfsec
-SHELL := /bin/bash
```

Comment:
This has implications for the github action which we need to resolve before removing from here